### ISSUE TITLE
Fix LICENCES in all repositories

### ISSUE THREAD

**Issue by MansMeg** [swerik-project/the-swedish-parliament-corpus#10]
@Lottabrorsson will look up and check what LICENCE we should have for the corpus and the persons database.

**Comment by lukavdplas** [comment]
What is the current status of this? Is there a timeline for when the corpus would be published with a re-usable licence?

By the way, the [project site](https://swerik-project.github.io/) suggests that the data has a CC-BY licencse, but the actual data repositories do not include a license statement. So the site should probably be corrected.

**Comment by MansMeg** [comment]
Yes. There is an ongoing discussion on what exact licence to use (cc-by, cc-zero or public domain) - we have need to look up exact what to use. @Lottabrorsson , maybe we should prioritize this now when the question was about it?
